numba_cuda/numba/cuda/cudadrv/driver.py

```diff
@@ -3639,3 +3624,20 @@ def inspect_obj_content(objpath: str):
             code_types.add(match.group(1))
 
     return code_types
+
+
+def _stream_handle(stream):
+    if stream == 0:
```

Comment:
Not specific to this PR change - but what does "0" mean for a stream?

Reply:
A small docstring in this helper is helpful in explaining the supported stream objects in Numba.

Reply:
This should just map to the default stream, I'll add docs